content/tutorials/android-push-notifications.textile

```diff
@@ -19,9 +22,13 @@ level: easy
 reading_time: 20
 languages:
   - android
+  - java
+  - nodejs
+  - javascript
 ably_product: push-notifications
 tags:
-  - ably-features
+  - Android
+  - Push notification
```

Comment:
Can we please replace all occurrences of the tag name 'Push notification' with 'Push Notifications'

Reply:
done (x2)